Makefile

```diff
@@ -133,11 +140,10 @@ doc: crdoc ## Generate
 	$(CRDOC) --resources config/crd/bases/flows.netobserv.io_flowcollectors.yaml --output docs/FlowCollector.md
 
 generate-go-conversions: $(CONVERSION_GEN) ## Run all generate-go-conversions
-		$(MAKE) clean-generated-conversions SRC_DIRS="./api/v1alpha1"
```

Comment:
pls add the cleanup back so we don't endup with stale auto generated conversion code

Reply:
Sure, done